### PR TITLE
Translate help messages to english

### PR DESCRIPTION
**Pull request by SwinX**

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -8,17 +8,17 @@ var _ = require('lodash'),
...
-        baseUrl: _.template('Базовый URL беты [${baseUrl}]')(defaults),
-        waitTimeout: _.template('Время ожидания событий на странице [${waitTimeout}]')(defaults),
-        screenshotPath: _.template('Путь для хранения скриншотов [${screenshotPath}]')(defaults),
-        reporters: _.template('Используемые репортеры [${reporters}]')(defaults),
-        debug: _.template('Включить дебаг webdriver [${debug}]')(defaults),
-        conf: _.template('Путь к конфигу [${conf}]')(defaults),
-        browser: 'Запуск тестов в определенном браузере'
+        grid: _.template('Selenium grid URL [${grid}]')(defaults),
+        baseUrl: _.template('Page under test base URL [${baseUrl}]')(defaults),
+        waitTimeout: _.template('Wait timeout for page events [${waitTimeout}ms]')(defaults),
```
Comment (sipayRT) [inline lib/cli.js]: Default timeout for all waitXXX commands

**Comment by SwinX** [comment]
:up:

**Comment by sipayRT** [comment]
ok

**Pull request merged**